docs/index.md

```diff
@@ -6,37 +6,47 @@ Project documentation with&nbsp;Markdown.
 
 ## Overview
 
-MkDocs is a **fast**, **simple** and **downright gorgeous** static site generator that's geared towards building project documentation. Documentation source files are written in Markdown, and configured with a single YAML configuration file.
+MkDocs is a **fast**, **simple** and **downright gorgeous** static site
+generator that's geared towards building project documentation. Documentation
+source files are written in Markdown, and configured with a single YAML
+configuration file.
 
----
-
-**MkDocs is currently still in development.**
+!!! warning
```

Comment:
Seems reasonable, perhaps we could just remove the message in general. It doesn't really give enough information to be helpful.